### PR TITLE
Fix wmts kvp server

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
I've tried to run the tests locally, but this is not documented. Is there a simple procedure to run tests locally ? I'd like to add some tests, but it is really boring to work with travis.

File: tilecloud_chain/server.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def __call__(self, environ, start_response):
     def serve(self, path, params, **kwargs):
         dimensions = []
 
-        if path is not None:
+        if path:
```
Comment (sbrunner) [inline tilecloud_chain/server.py]: Why should you do that?

Comment (arnaud-morvan) [inline tilecloud_chain/server.py]: In my project I use this :
```
config.add_route('tiles', '/tiles/\*path')
```
and path is not None but empty tuple.

Comment (sbrunner) [inline tilecloud_chain/server.py]: OK :-)

**Pull request merged**